src/main/routes/health.ts

```diff
@@ -6,17 +6,22 @@ import * as path from 'path'
 import { FeatureToggles } from 'utils/featureToggles'
 
 /* tslint:disable:no-default-export */
-export default express.Router()
-  .get('/health', healthcheck.configure({
-    checks: {
-      'claimstore': basicHealthCheck('claim-store'),
-      'draft-store': basicHealthCheck('draft-store'),
-      'fees': basicHealthCheck('fees'),
-      'pay': basicHealthCheck('pay'),
-      'idam-service-2-service-auth': basicHealthCheck('idam.service-2-service-auth'),
-      'idam-api': basicHealthCheck('idam.api')
-    }
-  }))
+
+let healthCheckRouter = express.Router()
```

Comment:
do we need this? looks like line 23 does it too...

Reply:
i did it as a middleware way